client/components/combobox.less

```diff
@@ -23,16 +27,26 @@
           border-radius: 10px;
           border: 3px solid #ffffff;
         }
-
-        .item {
-            position:relative;
+		&.open {
+			display: unset;
+		}
+		> * {
+			all: unset;
+			display: block;
+			width: 100%;
+			position:relative;
             font-size: 11px;
             font-family: Open Sans;
             padding: 5px;
             cursor: default;
             margin: 0 3px;
+			box-sizing: border-box;
```

Comment:
I believe our css applies this to all elements, consider changing `all:unset` if the approach is taking more lines than it should (you probably already have, but must suggest it)

Reply:
Largely the `all: unset` is to unset everything we have currently setup for *every* `button` on the site:

```css
button {
	-webkit-transition: background-color .25s ease;
	-moz-transition: background-color .25s ease;
	-o-transition: background-color .25s ease;
	transition: background-color .25s ease;
	display: inline-block;
	padding: .6em 1.2em;
	cursor: pointer;
	background-color: #27AE60;
	font-family: 'Open Sans',sans-serif;
	font-size: .8em;
	font-weight: 800;
	color: white;
	text-decoration: none;
	text-transform: uppercase;
	border: none;
	outline: none;
}
```

Probably it'd be ideal to change this styling to be based on *some* class, rather than styling every button with a very heavy-handed style.  As we start to use actual button elements rather than just `div` elements with `onClick` attributes, we are going to want to style those buttons in certain contexts ("navbar", "snippetbar", "viewbar", etc).

If I made that change in this PR, though, the PR would quickly spread to cover changes in many more files.  So probably left as it's own thing for now.  Instead, I'll just do `all: unset` and then build the style from the ground up.

Without putting much thought on it, I think the current "default button" styling should apply to the class name `call-to-action`, though I bet there are better names.